sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ied/IEDAdapterTest.java

```diff
@@ -173,23 +172,21 @@ void testMatches() throws Exception {
     }
 
     @Test
-    void createControlBlock() throws Exception {
+    void createControlBlock() {
         // Given
         SCL scd = SclTestMarshaller.getSCLFromFile("/ied-test-schema-conf/ied_unit_test.xml");
         SclRootAdapter sclRootAdapter = new SclRootAdapter(scd);
         IEDAdapter iedAdapter = assertDoesNotThrow(() -> sclRootAdapter.getIEDAdapterByName("IED_NAME"));
-        ReportControlBlock controlBlock = new ReportControlBlock();
-        controlBlock.setId("CBID1");
-        controlBlock.setName("rpt");
+        ReportControlBlock controlBlock = new ReportControlBlock("rpt", "CBID1", "dataSet");
         controlBlock.setConfRev(2L);
-        controlBlock.setDataSetRef("dataSet");
         controlBlock.setHolderIEDName("IED_NAME");
         controlBlock.setHolderLDInst("LD_INS2");
         controlBlock.setHolderLnClass("ANCR");
         controlBlock.setHolderLnInst("1");
         controlBlock.setOptFields(new TReportControl.OptFields());
         // When
-        iedAdapter.createControlBlock(controlBlock, ServiceSettingsType.GSE);
+        //FIXME
+        //iedAdapter.createControlBlock(controlBlock, ControlBlockEnum.GSE);
```

Comment:
???

Reply:
My bad, I forgot to delete this test. The method `createControlBlock` was moved to `AbstractLNAdapter`. I wrote new tests in LN0AdapterTest